### DISCUSSION TITLE
Where is Rocket benchmark ?

### DISCUSSION THREAD

**Discussion by MathieuSoysal** [TechEmpower/FrameworkBenchmarks#8767] [Q&A]
Hello, first thank you for this awesome project ! :D

I searched but I didn't found the rust web framework [Rocket](https://github.com/rwf2/Rocket) in the benchmark result : https://www.techempower.com/benchmarks/

Is it normal ?

Thank you for all !

**Comment by fakeshadow** [comment]
Yes it's normal. There are broken bench code keep wasting server resource so at some point some of them were disabled and rocket is one of such case. You can reference this PR where it's disabled https://github.com/TechEmpower/FrameworkBenchmarks/pull/8255

**Comment by joanhey** [comment]
It's broken the test for Rocket, please send a issue to the Rocket repository, to fix it.

Thank you.
